Dockerfile

```diff
@@ -1,4 +1,28 @@
+FROM mcr.microsoft.com/dotnet/sdk:3.1 as builder
+
+ARG GITHUBSHA OPENWEATHERMAP_APIKEY_RAW 
+
+WORKDIR /prelim-frontend
+
+ENV NODE_OPTIONS="--openssl-legacy-provider"
```

Comment:
Please add a comment (with a link to the node17 release docs) that this is due to the openssl 3.0 change. This way it's easier to check in the future whether we can remove this arg or not.